packages/slate-react/src/components/content.js

```diff
@@ -597,7 +597,7 @@ class Content extends React.Component {
       // COMPAT: In iOS, a formatting menu with bold, italic and underline
       // buttons is shown which causes our internal value to get out of sync in
       // weird ways. This hides that. (2016/06/21)
-      ...(readOnly ? {} : { WebkitUserModify: 'read-write-plaintext-only' }),
+      ...((readOnly || (!IS_IOS && !IS_SAFARI)) ? {} : { WebkitUserModify: 'read-write' }),
```

Comment:
```suggestion
      // [CC] https://github.com/ianstormtaylor/slate/issues/5110#issuecomment-1236831747
      ...(readOnly || (!IS_IOS && !IS_SAFARI) ? {} : { WebkitUserModify: 'read-write-plaintext-only' }),
```
- include the comment for context
- based on our current understanding, the value should be `'read-write-plaintext-only'` because the goal is to convince the affected browsers that the text should not be considered stylable.

Reply:
Oops, good catch,  I did that in the slate-editor but didn't match it within this slate project.